continuous_integration/install.sh

```diff
@@ -18,9 +18,9 @@ print_conda_requirements() {
     # if yes which version to install. For example:
     #   - for numpy, NUMPY_VERSION is used
     #   - for scikit-learn, SCIKIT_LEARN_VERSION is used
-    TO_INSTALL_ALWAYS="pip nose"
+    TO_INSTALL_ALWAYS="pip nose "
     REQUIREMENTS="$TO_INSTALL_ALWAYS"
-    TO_INSTALL_MAYBE="python numpy flake8"
+    TO_INSTALL_MAYBE="pytest python numpy flake8"
```

Comment:
I don't know the CI was unhappy initially. Once it became green, I opened a PR fr further discussions. Although I am still trying to settle at a proper version.